pygameweb/templates/error.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>{{ code }} - pygame.org</title>
+
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+
+    <style>
+        * {
+            font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+            text-align: center;
+            color: #111;
+        }
+
+        h1 {
+            margin-top: 128px;
+            font-size: 10em;
+            margin-bottom: 16px;
+        }
+
+        h2 {
+            font-size: 1.9em;
+            font-weight: 400;
+
+            padding: 0 64px;
+        }
+
+        h3 {
+            font-weight: 400;
+        }
+
+        p {
+            font-size: 1.2em;
+            color: #444;
+        }
+
+        a {
+            color: #337ab7;
+            text-decoration: none;
+        }
+
+        img {
+            height: 100px;
+        }
+    </style>
+
+</head>
+<body{% block body_attribs %}{% endblock body_attribs %}>
+
+<div id="container">
+    <img src="http://www.pygame.org/docs/pygame_logo.gif">
+    <h1>{{ code }}</h1>
+    <h2>{{ message }}</h2>
+    <h3>{{ sub }}</h3>
+    <p><a href="/">Home</a> - <a href="/docs">Docs</a> - <a target="_blank" href="https://github.com/pygame/pygameweb/issues/new">Report This</a></p>
```

Comment:
This line is a bit long to read.

Reply:
Nothing I can really do about that without really ugly line breaks between `<a` and `href=`. If I put the line breaks around the `-`s it'll remove the spaces around them.

Reply:
```html
     <p>
        <a href="/">Home</a>
        - <a href="/docs">Docs</a>
        - <a 
             target="_blank"
             href="https://github.com/pygame/pygameweb/issues/new">
             Report This
          </a>
     </p> 
```

Reply:
Semantically using a list in this situation feels wrong. Coupled with the fact that it would need a load of extra CSS to style it up it feel overkill. Breaking up the `<a` like that might look good though :)